iocore/eventsystem/I_Thread.h

```diff
@@ -138,18 +146,10 @@ class Thread
   ProxyAllocator ioAllocator;
   ProxyAllocator ioBlockAllocator;
 
-private:
-  // prevent unauthorized copies (Not implemented)
-  Thread(const Thread &);
-  Thread &operator=(const Thread &);
-
 public:
-  ink_thread start(const char *name, size_t stacksize, ThreadFunction f, void *a, void *stack);
+  ink_thread start(const char *name, void *stack, size_t stacksize, ThreadFunction const &f = ThreadFunction());
```

Comment:
That's the problem with splitting this up, there are pieces that are needed later but not directly applicable now.